src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java

```diff
@@ -347,8 +347,14 @@ public void clickSuEliminaPec() {
     public String waitLoadPopUpElimina() {
         By titlePopUp = By.id("dialog-title");
         By subTitlePopUp = By.id("dialog-description");
+        By cancelButton = By.id("buttonAnnulla");
+        By confirmButton = By.id("buttonConferma");
         this.getWebDriverWait(10).withMessage("Non è stato caricato il titolo del pop-up").until(ExpectedConditions.visibilityOfElementLocated(titlePopUp));
         this.getWebDriverWait(10).withMessage("Non è stato caricato il sottotitolo del pop-up").until(ExpectedConditions.visibilityOfElementLocated(subTitlePopUp));
+        this.getWebDriverWait(10).withMessage("Non è stato caricato il sottotitolo del pop-up").until(ExpectedConditions.visibilityOfElementLocated(cancelButton));
+        this.getWebDriverWait(10).withMessage("Non è stato caricato il titolo del pop-up").until(ExpectedConditions.visibilityOfElementLocated(confirmButton));
```

Comment:
Did you check if this update doesn't break any other test case?

Reply:
no , but every pop-up while deleting email has annulla and conderma button.